GOVERNANCE.md

```diff
@@ -0,0 +1,66 @@
+# Project Governance
+
+The Valkey project is managed by a Technical Steering Committee (TSC) composed of the maintainers of Valkey repository.
+The Valkey project includes all of the current and future repositories under the Valkey-io organization.
```

Comment:
"the Valkey-io organization" just means the github org. It sounds like were defining our project in terms of whats under the github org, but it's rather the other way around. Can we write it in a different way?

```suggestion
The Valkey project stores its repositories on GitHub under the valkey-io organization.
```

Reply:
I didn't want to tie the governance to specifically to github. Maybe I'll word it like:
```
The Valkey project includes all of the current and future repositories under the Valkey-io GitHub organization or whatever future code hosting platform the project uses.
```

?

Reply:
Sure. Or just keep it as is. It doesn't matter much.

Reply:
It's currently valkey-io organization. We're working on getting valkey though.